index.js

```diff
@@ -26,9 +26,16 @@ module.exports = function(options) {
         runOptions.push(file);
     });
 
-    runOptions.push('-f');
-    var format = options.format || 'pretty';
-    runOptions.push(format);
+    if (!options.format) {
+        options.format = 'pretty';
+    }
+
+    var formats = util.isArray(options.format) ? options.format : [options.format];
```

Comment:
yeah, that was because of someone's PR but please, take a look at this comment https://github.com/vgamula/gulp-cucumber/pull/27#issuecomment-181070452